packages/DraftEditor/stories/DraftEditor.stories.tsx

```diff
@@ -27,7 +27,7 @@ storiesOf("DraftEditor", module)
       };
 
       const preview = () => {
-        return parseTextBlock(draftContent);
+        return parseTextBlock({ editor: draftContent });
```

Comment:
You could refactor this into:

const preview = () => parseTextBlock({ editor: draftContent });